helm_deploy/prisoner-content-hub-frontend/templates/ingress.yaml

```diff
@@ -3,15 +3,22 @@
 apiVersion: networking.k8s.io/v1
 kind: Ingress
 metadata:
-  name: {{ $fullName }}
+  name: {{ $fullName }}-ingress
   labels:
     {{- include "prisoner-content-hub-frontend.labels" . | nindent 4 }}
   annotations:
     {{- range $k, $v := .Values.ingress.annotations }}
     {{ $k }}: {{ tpl $v (dict "qualifier" $.Values.ingress.qualifier "Template" $.Template) | quote }}
     {{- end}}
     nginx.ingress.kubernetes.io/whitelist-source-range: {{ include "app.joinListWithComma" .Values.ingress.allowed | quote }}
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+    {{ if eq .Values.environmentName "development" }}
+    external-dns.alpha.kubernetes.io/set-identifier: prisoner-content-hub-frontend-ingress-{{ .qualifier }}-prisoner-content-hub-development-green
+    {{ else }}
+    external-dns.alpha.kubernetes.io/set-identifier: prisoner-content-hub-frontend-ingress-prisoner-content-hub-{{ .Values.environmentName }}-green
+    {{ end }}
```

Comment:
This conditional is here to allow the set-identifier to work for the development environment (where it's a different format from staging/production).
This can be removed on the next PR, as we can revert back to adding set-identifier from the values-ENVIRONMENTNAME.yaml files.